Tests/LinuxMain.swift

```diff
@@ -1,7 +0,0 @@
-import XCTest
-
-import SwiftInspectorTests
-
-var tests = [XCTestCaseEntry]()
-tests += SwiftInspectorTests.allTests()
-XCTMain(tests)
```

Comment:
We're not using this file, and if we needed to (in order to run this project in linux) we should generate this in CI instead of manually maintaining this.

Reply:
Why do you think we should generate it in CI?

Reply:
@fdiaz ^